.github/workflows/tests.yml

```diff
@@ -212,6 +215,13 @@ jobs:
 
       - uses: codecov/codecov-action@e156083f13aff6830c92fc5faa23505779fbf649 # v1.2.1
 
+      - name: Cache parallel tests log
```

Comment:
This isn't being restored from the cache before the tests are run so won't actually be doing anything.

Reply:
Want to do this on macOS, too.